docs/sources/contribute/documentation.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Documentation Guidelines"
+description = "Documentation Guidelines"
+type = "docs"
+[menu.docs]
+parent = "contribute"
+weight = 2
++++
+
+# Documentation Guidelines
+
+All Grafana documentation is written using [Markdown](https://en.wikipedia.org/wiki/Markdown), and can be found in the [docs](https://github.com/grafana/grafana/tree/master/docs) directory in the [Grafana GitHub repository](https://github.com/grafana/grafana). The [documentation website](https://docs.grafana.com.)  is generated with [hugo](https://gohugo.io) which uses [Blackfriday](https://github.com/russross/blackfriday) as its Markdown rendering engine.
+
+## Structure
+
+The documentation is organized into topics, called _sections_.
+
+Each top-level section is located under the [docs/sources](https://github.com/grafana/grafana/tree/master/docs/sources) directory. Subsections are added by creating a subdirectory in the directory of the parent section.
+
+For each section, a `_index.md` file is used to provide an overview of the topic.
+
+## Your first contribution
+
+If you’re unsure about where to start, check out some of our [open docs issues](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3Atype%2Fdocs).
+
+When you’ve found an issue you want to work on, you’re encouraged to comment on the issue to let other people know you intend to work on it.
+
+## Community
+
+If you have questions on a specific issue, post a comment to ask for clarification, or to give feedback.
+
+For general discussions on documentation, you’re welcome to join the #docs channel on our [public Grafana Slack](http://slack.raintank.io) team.
```

Comment:
I would post this before structure.

Organization of doc something like:

What I need to know before I start (structure matters after I find an issue to work on)
How do I start
Who do I ask if I am confused
Project information, like structure, style rules, other guidelines